File: Web/rishu15_HrishikDwivedi_2125ec1006_02/gettingStarted/primeTillN.cpp
#include <iostream>
using namespace std;
int main(int argc, char **argv){
    int low, high;
    cin >> low >> high;

    //write your code here
    for(int i = low;i<=high;i++){
        int count=0;
        if(i==0||i==1){
            count=1;
        }
        for(int j=2;j<=i/2;j++){
            if(i%j==0){
                count++;
                break;
            }
        }
        if(count==0){
            cout<<i<<endl;
        }
    }

}

File: Android/Cyclotron17_SambhrantTiwari_2125csai1011_2/GettingStarted/PrimeTillN.java
import java.util.*;

public class Main{
    public static void main(String[] args) {
        // write your code here

      int k,c=0;
      long high,i,low,j;
      Scanner scn = new Scanner(System.in);
      low=scn.nextLong();
      high=scn.nextLong();
      
  
       // write ur code here
       for(i=low;i<=high;i++)
       {
             c=0;
            
             for(j=1;j<=i;j++)
             {
                 if(i%j==0)
                 {
                    c++; 
                 }
               
             }
             if(c==2)
             {
                System.out.println(i);
             }


             

       }
  
   }
}
  
    
